tests/unit/test_orgs_invites.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from github3 import GitHubError
+from github3.orgs import Organization
+
+from . import helper
+
+url_for = helper.create_url_helper('https://api.github.com/orgs/github')
+
+get_org_example_data = helper.create_example_data_helper(
+    'org_example')
+
+
+class TestInvites(helper.UnitIteratorHelper):
+    described_class = Organization
+    example_data = get_org_example_data()
+
+    def test_get_invitations_iterator(self):
```

Comment:
Same as above. We name test methods after what they're testing and this one belongs on `TestOrganization`.